docs/en/enterprise-edition/policy-reference/sast-policies/go-policies/sast-policy-259.adoc

```diff
@@ -0,0 +1,70 @@
+== Use of profiling endpoint in production
```

Comment:
- Profiling endpoint used in production
- Usage of profiling endpoint in production